doc/classes/MeshInstance3D.xml

```diff
@@ -67,7 +67,7 @@
 		</method>
 		<method name="get_active_material" qualifiers="const">
 			<return type="Material" />
-			<param index="0" name="surface" type="int" />
+			<param index="0" name="surface" type="int" default="0" />
 			<description>
 				Returns the [Material] that will be used by the [Mesh] when drawing. This can return the [member GeometryInstance3D.material_override], the surface override [Material] defined in this [MeshInstance3D], or the surface [Material] defined in the [member mesh]. For example, if [member GeometryInstance3D.material_override] is used, all surfaces will return the override material.
```

Comment:
The description would have to be reworded entirely as fitting this detail is a bit awkward, but it's better to give some additional attention to the parameter itself.
```suggestion
				Returns the [Material] for the specified [param surface] that will be used by the [Mesh] when drawing. This can return the [member GeometryInstance3D.material_override], the surface override [Material] defined in this [MeshInstance3D], or the surface [Material] defined in the [member mesh]. For example, if [member GeometryInstance3D.material_override] is used, all surfaces will return the override material.
```